content/rs/security/tls-ssl.md

```diff
@@ -6,68 +6,82 @@ alwaysopen: false
+1. Optional: To limit connections further to a subset of those with valid certificates, enforce **Subject Alternative Name** and enter authorized users separated with commas.
+1. Select **Update** at the bottom of the screen to save your configuration.
 
-{{< note >}}
-There are two considerations for replication authentication you should be aware of:
+## Enable TLS for Active-Active cluster connections
 
-1. The syncer certificates of the clusters that host the replica instances of the database must always be set when enabling a database for encryption.
-2. When using Active-Active, the syncer certificate for each cluster must be configured on the database.
+{{< note >}}
+You cannot enable or disable TLS after the Active-Active database is created, but you can change the TLS configuration.
 {{< /note >}}
 
-## Certificate Authentication for Active-Active Databases
+### Retrieve syncer certificates
 
-When you create a new CRDB, you can configure authentication for traffic between active-active databases using the same process for as replication traffic.
+1. For each participating cluster, copy the syncer certificate from the **general** settings tab.
 
-{{< note >}}
-You cannot enable or disable TLS after the CRDB is created, but you can change
-the TLS configuration.
-{{< /note >}}
+### Configure TLS certificates for Active-Active
 
-### Configuring TLS for CRDB communication
+1. During database creation (see [Create an Active-Active Geo-Replicated Database]({{<relref "content/rs/administering/creating-databases/create-active-active.md" >}}), select **Edit** from the **configuration** tab.
+1. Enable **TLS**.
+    - **Enforce client authentication** is selected by default. If you choose to change this option, you will enforce encryption without authentication.
+1. Select **Require TLS for CRDB communication only** from the dropdown menu.
+    ![crdb-tls-all](/images/rs/crdb-tls-all.png "crdb-tls-all")
+1. Select **Add** ![Add](/images/rs/icon_add.png#no-click "Add")
+1. Paste a syncer certificate into the text box.
+    ![database-tls-replica-certs](/images/rs/database-tls-replica-certs.png "Database TLS Configuration")
+1. Save the syncer certificate. ![icon_save](/images/rs/icon_save.png#no-click "Save")
+1. Repeat this process, adding the syncer certificate for each participating cluster.
+1. Optional: If also you want to  require TLS for client connections, select **Require TLS for All Communications** from the dropdown and add client certificates as well.
+1. Select **Update** at the bottom of the screen to save your configuration.
+
+### Configure TLS on all participating clusters
+
+Repeat this process on all participating clusters.
 
-To enable TLS for CRDB communication for a CRDB:
+To enforce TLS authentication, Active-Active databases require syncer certificates for every cluster connection. If every cluster does not have the syncer certificate for every other participating cluster, it will cause read and write errors.
```

Comment:
```suggestion
To enforce TLS authentication, Active-Active databases require syncer certificates for every cluster connection. If every participating cluster does not have a syncer certificate for every other participating cluster, read and write errors will occur.
```

Reply:
I would suggest: "If every participating cluster does not have a syncer certificate for every other participating cluster, synchronization will fail"
Or something similar. What will happen is the clusters will not even be able to establish the (tls) connection. It will prevent synchronization between the two (or more) clusters w/o the certs, but quickly also affect the "whole" AA DB since it's "waiting" for the isolated replica to communicate.
I hope that was clear